.github/workflows/build.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         run: tests/scripts/validate_modified_files.sh build
 
       - name: run codegen
-        run: GOPATH=$(go env GOPATH) make codegen
+        run: GOPATH=$(go env GOPATH) make gen.codegen
```

Comment:
`gen.codegen` is redundant, can we just keep `codegen`?

Reply:
@travisn  wrote:
> `gen.codegen` is redundant, can we just keep `codegen`?

I share your concern with this being redundant, but I was following what @BlaineEXE  suggested in #14686 

If this is a formal change request, I can undo this change.

Reply:
@travisn  these changes are all gone from this PR now.